plugins/vppv2/ifplugin/descriptor/interface.go

```diff
@@ -297,6 +301,15 @@ func (d *InterfaceDescriptor) equivalentMemifs(oldMemif, newMemif *interfaces.Me
 	return true
 }
 
+// equivalentIPSecTunnels compares two IPSec tunnels for equivalence.
+func (d *InterfaceDescriptor) equivalentIPSecTunnels(oldTun, newTun *interfaces.IPSecLink) bool {
+	return oldTun.Esn == newTun.Esn && oldTun.AntiReplay == newTun.AntiReplay && oldTun.LocalSpi == newTun.LocalSpi &&
```

Comment:
Please compare single field per line for better readability of which fields are being compared.